### PR TITLE
installation: fix invenio_base imports

### PR DESCRIPTION
**Pull request by SamiHiltunen**
* FIX Adds missing `invenio_base` dependency and amends past
  upgrade recipes following its separation into standalone package.

Signed-off-by: Sami Hiltunen <sami.mikael.hiltunen@cern.ch>

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -57,7 +57,7 @@ install:
   - "inveniomanage config create secret-key"
   - "inveniomanage config set CFG_EMAIL_BACKEND flask_email.backends.console.Mail"
   - "inveniomanage config set CFG_BIBSCHED_PROCESS_USER `whoami`"
-  - "inveniomanage config set PACKAGES ['invenio.base', 'invenio.modules.accounts', 'invenio.modules.records', 'invenio_tags']"
+  - "inveniomanage config set PACKAGES ['invenio_base', 'invenio.modules.accounts', 'invenio.modules.records', 'invenio_tags']"
```
Comment (jirikuncar) [inline .travis.yml]: can you please update `.travis.yml` based on `invenio-accounts` and and `.travis.invenio.cfg` with current configuration?

Please note that we need to update 'invenio.modules.accounts' and 'invenio.modules.records' too.

File: setup.py [hunk 2]
```diff
@@ -31,6 +31,7 @@
 requirements = [
     'Flask',
     'six',
+    'invenio-base>=0.1.0',
```
Comment (jirikuncar) [inline setup.py]: we might have more dependencies missing ...

File: tests/test_tags_restful.py [hunk 3]
```diff
@@ -42,7 +42,7 @@ def config(self):
         """Config."""
         cfg = super(TestTagsRestfulAPI, self).config
         cfg['PACKAGES'] = [
-            'invenio.base',
+            'invenio_base',
```
Comment (jirikuncar) [inline tests/test_tags_restful.py]: @SamiHiltunen can you amend also the rest of packages?

The order should be:
```
invenio_tags
invenio_oauth2server
invenio_accounts
invenio_base
```

File: .travis.yml [hunk 4]
```diff
@@ -57,7 +57,7 @@ install:
   - "inveniomanage config create secret-key"
   - "inveniomanage config set CFG_EMAIL_BACKEND flask_email.backends.console.Mail"
   - "inveniomanage config set CFG_BIBSCHED_PROCESS_USER `whoami`"
-  - "inveniomanage config set PACKAGES ['invenio.base', 'invenio.modules.accounts', 'invenio.modules.records', 'invenio_tags']"
+  - "inveniomanage config set PACKAGES ['invenio_base', 'invenio_accounts', 'invenio.modules.records', 'invenio_tags']"
```
Comment (jirikuncar) [inline .travis.yml]: invenio_tags, invenio_records, invenio_accounts, invenio_base

File: docs/_ext/flask_app.py [hunk 5]
```diff
@@ -23,8 +23,8 @@
 def setup(sphinx):
     """Setup Sphinx object."""
     from flask import has_app_context
-    from invenio.base.factory import create_app
-    PACKAGES = ['invenio.base', 'invenio.modules.accounts',
+    from invenio_base.factory import create_app
+    PACKAGES = ['invenio_base', 'invenio.modules.accounts',
```
Comment (jirikuncar) [inline docs/_ext/flask_app.py]: invenio_tags, invenio_records, invenio_accounts, invenio_base

File: setup.py [hunk 6]
```diff
@@ -31,6 +31,7 @@
 requirements = [
     'Flask',
     'six',
+    'invenio-base>=0.2.1',
     'invenio-upgrader>=0.1.0',
     'Invenio',
```
Comment (jirikuncar) [inline setup.py]: please remove invenio and add invenio_records, invenio_accounts, invenio_oauth2server

**Comment by SamiHiltunen** [comment]
@jirikuncar This is most likely ready now.

File: .travis.yml [hunk 7]
```diff
@@ -57,7 +57,7 @@ install:
   - "inveniomanage config create secret-key"
   - "inveniomanage config set CFG_EMAIL_BACKEND flask_email.backends.console.Mail"
   - "inveniomanage config set CFG_BIBSCHED_PROCESS_USER `whoami`"
-  - "inveniomanage config set PACKAGES ['invenio.base', 'invenio.modules.accounts', 'invenio.modules.records', 'invenio_tags']"
+  - "inveniomanage config set PACKAGES ['invenio_tags', 'invenio_records', 'invenio_accounts', 'invenio_base']"
```
Comment (jirikuncar) [inline .travis.yml]:
```
2.34s$ inveniomanage config set PACKAGES ['invenio_tags', 'invenio_records', 'invenio_accounts', 'invenio_base']
usage: inveniomanage [-?]
                     {shell,knowledge,database,clean,check-for-software-updates,runserver,records,version,bower,upgrader,collect,apache,detect-system-details,formatter,config,show-urls,assets}
                     ...
inveniomanage: error: too many arguments
```

you need to better escape the arguments or use `.kwalitee.invenio.cfg` trick from other repositories.

File: .travis.invenio.cfg [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+PACKAGES = [
+    'invenio_accounts',
+    'invenio_groups',
+    'invenio_oauth2server',
+    'invenio_records',
+    'invenio_search',
+    'invenio_tags',
+    'invenio_upgrader',
+    'invenio_base',
+]
```
Comment (jirikuncar) [inline .travis.invenio.cfg]: please add `\n` but not `\n\n` ;)

File: .travis.yml [hunk 9]
```diff
@@ -17,78 +17,52 @@
...
 
 script:
-  - "sphinx-build -qnNW docs docs/_build/html"
+  - "sphinx-build -qnN docs docs/_build/html"
   - "python setup.py test"
-#  - "wget -O /dev/null http://localhost"
 
 after_success:
-  - coveralls
+  - coveralls
```
Comment (jirikuncar) [inline .travis.yml]: missing `\n`

File: docs/_ext/flask_app.py [hunk 10]
```diff
@@ -23,9 +23,9 @@
 def setup(sphinx):
     """Setup Sphinx object."""
     from flask import has_app_context
-    from invenio.base.factory import create_app
-    PACKAGES = ['invenio.base', 'invenio.modules.accounts',
-                'invenio.modules.records', 'invenio_tags']
+    from invenio_base.factory import create_app
+    PACKAGES = ['invenio_tags', 'invenio_records',
+                'invenio_accounts', 'invenio_tags']
```
Comment (jirikuncar) [inline docs/_ext/flask_app.py]: `'invenio_accounts', 'invenio_base']`

**Comment by jirikuncar** [comment]
@SamiHiltunen please close it if everything was done in #12.

**Pull request closed without merging**